lib/broccoli/ember-app.js

```diff
@@ -291,7 +294,11 @@ EmberApp.prototype._initVendorFiles = function() {
     jquery           = ember.paths.jquery;
   } else {
     jquery = this.bowerDirectory + '/jquery/dist/jquery.js';
-    emberShims = this.bowerDirectory + '/ember-cli-shims/app-shims.js';
+
+    if (emberCliShimsFromBower) {
+      emberShims = this.bowerDirectory + '/ember-cli-shims/app-shims.js';
```

Comment:
path would guarantee posix styles paths, no? totally understand if you don't want to change as a part of this PR though.